tasks/collect-data/collect-data.yaml

```diff
@@ -76,45 +76,46 @@ spec:
           value: '$(params.snapshot)'
       script: |
         #!/usr/bin/env sh
+        # shellcheck disable=SC3040
```

Comment:
Rather than this, I would suggest using `bash` instead of `sh` above.

Reply:
Thanks. I'll change to use `bash` .